lumigator/frontend/src/services/experiments/api.js

```diff
@@ -1,4 +1,8 @@
-export const PATH_EXPERIMENTS_EVALUATE = () => `jobs/evaluate/`
-export const PATH_EXPERIMENTS_ROOT = () => `health/jobs/`
-export const PATH_EXPERIMENT_DETAILS = (experiment_id) => `health/jobs/${experiment_id}`
+export const PATH_EXPERIMENTS_ROOT = () => `health/jobs/`;
+export const PATH_EXPERIMENT_DETAILS = (experiment_id) => `health/jobs/${experiment_id}`;
+export const PATH_EXPERIMENT_LOGS = (id) => `health/jobs/${id}/logs`
 // these 👆 endopoints do not belong to the experiments API,
+// temporary solution until
+// migration to "job" API route
+export const PATH_EXPERIMENT_RESULTS = (job_id) => `jobs/${job_id}/result/download`
+export const PATH_EXPERIMENTS_EVALUATE = () => `jobs/evaluate/`
```

Comment:
Just a quick note that the preferred interface (and the one that should remain stable for the MVP and will handle the required combinations of jobs under the hood) is `/experiments`. 

Using `/jobs` forces the UI to make workflow decisions (the temporary need to do so was fixed with #377). 

Give it a spin and shout if there is an issue! :)

@agpituk

Reply:
Results and evaluation(triggering experiment) were update.
Remaining issues are: 

1. `/experiments` doesn't provide the **RUNNING** status
2. `/experiments/{experiment_id}` is missing
     🔎 Used dataset
     🔎 Tested model
     🔎 Use case
     🔎 Number of samples

3. `/experiments/{experiment_id}/results` returns 404